app/Controllers/userController.php

```diff
@@ -211,9 +211,20 @@ public function manageAction() {
 		}
 	}
 
-	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfig = array(), $insertDefaultFeeds = true) {
-		if (!is_array($userConfig)) {
-			$userConfig = array();
+	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfigOverride = array(), $insertDefaultFeeds = true) {
+		$userConfig = array();
+
+		$customUserConfigPath = join_path(DATA_PATH, 'config-user.custom.php');
+		if (file_exists($customUserConfigPath)) {
+			$customUserConfig = include($customUserConfigPath);
```

Comment:
`@include()` and testing for `!= false` would catch more cases, e.g. no access rights, race condition...

Reply:
Ah, that was maybe for a comment I remember to have seen earlier about errors in the included file. Ok.